<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1710</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="graphPage">
       <widget class="QLabel" name="dataLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Graph Page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="dataPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="graphLabel">
          <property name="text">
           <string>test</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1710</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="toolDock">
    <widget class="QListWidget" name="toolList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>851</height>
      </rect>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <item>
      <property name="text">
       <string>Import Data</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PC Search</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GFCI Search</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FGES Search</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="helpAction"/>
   <addaction name="tutorialsAction"/>
   <addaction name="exportGraphAction"/>
   <addaction name="exportEdgesAction"/>
   <addaction name="exportComparisonsAction"/>
  </widget>
  <action name="helpAction">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="tutorialsAction">
   <property name="text">
    <string>Tutorials</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="exportGraphAction">
   <property name="text">
    <string>Export Graph</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="exportEdgesAction">
   <property name="text">
    <string>Export Edges</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="exportComparisonsAction">
   <property name="text">
    <string>Export Comparisons</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
